src/containers/NoteForm/NoteForm.js

```diff
@@ -49,40 +54,55 @@ export class NoteForm extends Component {
     e.preventDefault()
     const { id, title } = this.state.note;
     const { items } = this.state;
-    const url = 'http://localhost:3001/api/v1/notes'
-    this.props.postNote(url, { id, title, items })
-    this.setState({
-      redirect: true
-    })
+    const { isEdit, editNote, postNote } = this.props;
+    const url = isEdit ? `http://localhost:3001/api/v1/notes/${id}` : 'http://localhost:3001/api/v1/notes'
```

Comment:
This conditional logic looks really clean. Really like how you handled swapping the URL based on what action we are intending to perform!